site/standalone/setup.md

```diff
@@ -3,72 +3,23 @@ title: Setup Weave Flux Manually
 menu_order: 20
 ---
 
-# Connecting fluxctl to the daemon
+# Detailed Description
```

Comment:
Should we title this file "advanced configuration" or "customising the deployment" or something?

Reply:
I think this needs a rewrite, it's not clear how it overlaps with the other instructions. Doesn't need to happen in this PR though.